core/commands/snapshot_command.rb

```diff
@@ -272,6 +272,18 @@ def ntp_service_name(node_name)
     (box.downcase =~ /(ubuntu|debian)/).nil? ? 'ntpd' : 'ntp'
   end
 
+  def sync_node_time_command(node_name)
+    box = @config.template[node_name]['box']
+    ((box.downcase =~ /(rhel_6|centos_6)/).nil? ? 'sntp -s' : 'ntpdate') + ' 0.europe.pool.ntp.org'
+  end
+
+  def change_service_state(node_name, ntp_service, state)
```

Comment:
Add comments to the method. Also the name `change_service_state` is wrong as it states that the service state will be changed. But method only generates the statement to change state.